File: gems/decomplex/examples/zig/function-lcom.zig
pub fn mixed(price: i32, tax: i32, logger: Logger) Result {
    const subtotal = price + tax;
    const total = subtotal * 2;
    const rounded = total.round();

    const timestamp = now();
    var buffer = Buffer.init();
    buffer.push(timestamp);
    logger.info(buffer);

    return Result.init(rounded, buffer);
}

File: gems/decomplex/examples/rust/function-lcom.rs
fn mixed(price: i32, tax: i32, logger: Logger) -> (i32, Buffer) {
    let subtotal = price + tax;
    let total = subtotal * 2;
    let rounded = total.round();

    let timestamp = now();
    let mut buffer = Buffer::new();
    buffer.push(timestamp);
    logger.info(buffer);

    (rounded, buffer)
}
